CSharp/Library/Microsoft.Bot.Connector.Shared/ActionTypes.cs

```diff
@@ -47,5 +47,10 @@ public class ActionTypes
         /// Signin button
         /// </summary>
         public const string Signin = "signin";
+
+        /// <summary>
+        /// Invoke a command back to the bot
+        /// </summary>
+        public const string Invoke = "invoke";
```

Comment:
from https://msdn.microsoft.com/en-us/microsoft-teams/botsmessages#action---invoke-new] as highlighted in a [confused StackOverflow user's post](http://stackoverflow.com/questions/43623385/cardaction-button-with-type-invoke-doesnt-work-in-microsoft-bot-framework)

![image](https://cloud.githubusercontent.com/assets/20270743/25541054/1cec59ae-2c02-11e7-87a9-65809a3d96e8.png)